meta-resin-common/recipes-connectivity/resin-ntp-config/resin-ntp-config/resin-ntp-config.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Resin NTP configuration service
+DefaultDependencies=no
+Requires=mnt-boot.mount etc-systemd-timesyncd.conf.mount
+After=mnt-boot.mount etc-systemd-timesyncd.conf.mount
+Before=systemd-timesyncd.service
+
+[Service]
+ExecStart=@BASE_BINDIR@/bash @BINDIR@/resin-ntp-config
+Type=oneshot
+RemainAfterExit=yes
+StandardError=journal+console
```

Comment:
I'm referring to StandardError which by default should be on `inherit`. I'm not sure why we have it in net-config either - most probably by mistake. I remember cleaning some of the service in the past maybe I missed this one unless there is a reason that I can't remember of.